packages/theme/components/CartSidebar.vue

```diff
@@ -280,7 +291,7 @@ export default defineComponent({
     const { isAuthenticated } = useUser();
     const { send: sendNotification, notifications } = useUiNotification();
 
-    const products = computed(() => cartGetters.getItems(cart.value));
+    const products = computed(() => cartGetters.getItems(cart.value).filter(Boolean));
```

Comment:
You have already filtered items in composable
```suggestion
    const products = computed(() => cartGetters.getItems(cart.value);
```

Reply:
Sometimes there is null inside, I was not able to track down where is the source.